backend/src/main/java/gov/cdc/usds/simplereport/config/SimpleReportProperties.java

```diff
@@ -0,0 +1,22 @@
+package gov.cdc.usds.simplereport.config;
+
+import java.util.List;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.ConstructorBinding;
+
+
+@ConfigurationProperties(prefix = "simple-report")
```

Comment:
Probably want to have individual properties classes be one level down from here, otherwise this is going to be an awfully big object (Tom and I are also working underneath this prefix right now).

I think you could actually just bind an arraylist to "simple-report.admin-emails", like this:

    @ConfigurationProperties(prefix="simple-report.admin-emails")
    public class AdminEmailList extends ArrayList<String> { /* empty class declaration except for serialVersionId */ }

I readily admit that this is kind of cheeky.